ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1971,6 +1971,12 @@ router_multicast:
     routing: < true | false >
     multipath: < none | deterministic | "deterministic color" | "deterministic router-id" >
     software_forwarding: < kernel | sfe >
+    rpf:
+      routes:
+        - subnet: < IPv4 subnet / netmask >
+          destinations:
+            - nexthop: < ip | interface_name >
```

Comment:
Please add some context over this line like below:

```
destinations:
  # comment
  - nexthop: <>
 ```

Also, change the values in nexthop to ```< nexthop_ip | interface_name >```

Reply:
I think by just adding the value as suggested, we don't need the description.